src/SprykerEco/Yves/Computop/Handler/ExpressCheckout/ComputopPayPalExpressInitHandler.php

```diff
@@ -0,0 +1,174 @@
+ */
+
+namespace SprykerEco\Yves\Computop\Handler\ExpressCheckout;
+
+use Generated\Shared\Transfer\ComputopPayPalExpressPaymentTransfer;
+use Generated\Shared\Transfer\QuoteTransfer;
+use Spryker\Client\Shipment\ShipmentClientInterface;
+use Spryker\Shared\Kernel\Transfer\TransferInterface;
+use SprykerEco\Client\Computop\ComputopClientInterface;
+use SprykerEco\Shared\Computop\ComputopConfig;
+use SprykerEco\Yves\Computop\ComputopFactory;
+use SprykerEco\Yves\Computop\Converter\ConverterInterface;
+use SprykerEco\Yves\Computop\Dependency\Client\ComputopToQuoteClientInterface;
+use SprykerEco\Yves\Computop\Mapper\Init\PrePlace\PayPalExpressToQuoteMapperInterface;
+
+class ComputopPayPalExpressInitHandler implements ComputopPayPalExpressInitHandlerInterface
+{
+    /**
+     * @var \SprykerEco\Yves\Computop\Converter\ConverterInterface
+     */
+    protected $converter;
+
+    /**
+     * @var \SprykerEco\Yves\Computop\Dependency\Client\ComputopToQuoteClientInterface
+     */
+    protected $quoteClient;
+
+    /**
+     * @var \SprykerEco\Client\Computop\ComputopClientInterface
+     */
+    protected $computopClient;
+
+    /**
+     * @var \Spryker\Client\Shipment\ShipmentClientInterface
+     */
+    protected $shipmentClient;
+
+    /**
+     * @var \SprykerEco\Yves\Computop\Mapper\Init\PrePlace\PayPalExpressToQuoteMapperInterface
+     */
+    protected $payPalExpressToQuoteMapper;
+
+    /**
+     * @param \SprykerEco\Yves\Computop\Converter\ConverterInterface $converter
+     * @param \SprykerEco\Yves\Computop\Dependency\Client\ComputopToQuoteClientInterface $quoteClient
+     * @param \SprykerEco\Client\Computop\ComputopClientInterface $computopClient
+     * @param \Spryker\Client\Shipment\ShipmentClientInterface $shipmentClient
+     * @param \SprykerEco\Yves\Computop\Mapper\Init\PrePlace\PayPalExpressToQuoteMapperInterface $payPalExpressToQuoteMapper
+     */
+    public function __construct(
+        ConverterInterface $converter,
+        ComputopToQuoteClientInterface $quoteClient,
+        ComputopClientInterface $computopClient,
+        ShipmentClientInterface $shipmentClient,
+        PayPalExpressToQuoteMapperInterface $payPalExpressToQuoteMapper
+    ) {
+        $this->converter = $converter;
+        $this->quoteClient = $quoteClient;
+        $this->computopClient = $computopClient;
+        $this->shipmentClient = $shipmentClient;
+        $this->payPalExpressToQuoteMapper = $payPalExpressToQuoteMapper;
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     * @param array $responseArray
+     *
+     * @return \Generated\Shared\Transfer\QuoteTransfer
+     */
+    public function handle(QuoteTransfer $quoteTransfer, array $responseArray): QuoteTransfer
+    {
+        $responseTransfer = $this->converter->getResponseTransfer($responseArray);
+        $this->addPaymentToQuote($quoteTransfer, $responseTransfer);
```

Comment:
Please separate a logic block of code with a new line, it increases the code readability.